docs/developers/plugins.md

```diff
@@ -2,16 +2,20 @@
 
 Plugins are the most efficient way to customize or change the default behavior of a chart. They have been introduced at [version 2.1.0](https://github.com/chartjs/Chart.js/releases/tag/2.1.0) (global plugins only) and extended at [version 2.5.0](https://github.com/chartjs/Chart.js/releases/tag/2.5.0) (per chart plugins and options).
 
-## Popular Plugins
+## Popular Chart Type Plugins
```

Comment:
I updated the PR to put it in between where we list all the built-in charts and the popular plugins. It'd make sense to link to this section from charts.md, but I'm not quite sure how to do that without digging in a bit more